### DISCUSSION TITLE
7940 / Wireguard / UDP

### DISCUSSION THREAD

**Discussion by Greg57070** [Freetz-NG/freetz-ng#365] [Allgemein]
I'm facing a weird issue on 7940 with wireguard and UDP packets :
I can't transmit UDP packet through wireguard from computer on lan (but I can receive with no problem)
From the fritzbox itself, it's also work.

On 7530 or 7583, this bug is not reproducible.

The most amazing, if I start html/capture on the fritzbox, the problem disappear...

The problem is easily reproducible with iperf3 -u (client on fritzbox lan side)

Thanks in advance,
Gregory

**Comment by fda77** [comment]
It's well known the fritzos network stack is crap. It can even not handle to answer pings of 2 different hosts at the same time!
You could try to disable "packet accelerator" (avm pa), see faq. Network throughtput is then much slower, but maybe eought for your internet connection
